package.json

```diff
@@ -37,6 +37,7 @@
   "dependencies": {},
   "devDependencies": {
     "@angular/compiler": "^8.0.3",
+    "@angular/common": "^8.0.3",
```

Comment:
I think you missed to commit package-lock.json. That is why the build failed. 
This file must be include before a merge.